tests/openmetrics/test_exposition.py

```diff
@@ -94,6 +95,125 @@ def test_histogram(self):
 # EOF
 """, generate_latest(self.registry))
 
+
+    def test_native_histogram(self):
+        hfm = HistogramMetricFamily("nh", "nh")
+        hfm.add_sample("nh", None, None, None, None, NativeHistogram(24, 100, 0, 0.001, 4, (BucketSpan(0, 2), BucketSpan(1, 2)), (BucketSpan(0, 2), BucketSpan(1, 2)), (2, 1, -3, 3), (2, 1, -2, 3)))
```

Comment:
The type annotation of add_sample says `labels: Dict[str, str]`, which means that `None` is not allowed. I think adding `{}` should work. Same comment on the rest of the tests.

The type annotation of add_sample says ` value: float` , which means that `None` is not allowed. Also valid comment for the other tests here.

```suggestion
        hfm.add_sample("nh", None, 0, None, None, NativeHistogram(24, 100, 0, 0.001, 4, (BucketSpan(0, 2), BucketSpan(1, 2)), (BucketSpan(0, 2), BucketSpan(1, 2)), (2, 1, -3, 3), (2, 1, -2, 3)))
```

The reason why `mypy` CI check doesn't catch these is due to not having type annotation on the `def` line.